### PR TITLE
Added support for Discord notifications

### PR DESCRIPTION
**Pull request by jakest**
Added support for sending notifications to Discord. Also updated the documentation with instructions.

### CODE REVIEW

File: setup/pi/setup-teslausb [hunk 1]
```diff
@@ -46,6 +46,8 @@ function read_setup_variables {
   newnamefor[ifttt_enabled]=IFTTT_ENABLED
   newnamefor[ifttt_event_name]=IFTTT_EVENT_NAME
   newnamefor[ifttt_key]=IFTTT_KEY
+  newnamefor[discord_enabled]=DISCORD_ENABLED
+  newnamefor[discord_webhook_url]=DISCORD_WEBHOOK_URL
```
Comment (marcone) [inline setup/pi/setup-teslausb]: Don't do this.The "newnamefor" stuff is for backwards compatibility with older config files, so this isn't needed here.

File: run/send-push-message [hunk 2]
```diff
@@ -23,6 +23,14 @@ function send_gotify () {
     "$GOTIFY_DOMAIN/message?token=$GOTIFY_APP_TOKEN"
 }
 
+function send_discord() {
+  log "Sending Discord message."
+
+  curl -H "Content-Type: application/json" -d \
+    '{"username": "TeslaUSB", "content": "'"$message"'"}' \
+    $DISCORD_WEBHOOK_URL
```
Comment (marcone) [inline run/send-push-message]: shellcheck wants double quotes around the url var

Comment (jakest) [reply run/send-push-message]: @marcone added

**Comment by jakest** [comment]
@marcone the requested changes have been made. Thanks for pointing out the backwards compatibility variables.

**Pull request merged**